bin/mmdb_utils.py

```diff
@@ -241,23 +284,31 @@ def update_lookup(self):
                 method='POST', raiseAllErrors=True)
 
 
-    def download_mmdb_database(self, account_id, license_key):
-        # NOTE - Proxy Configuration
-        # Remove '<username>:<password>@' part if using proxy without authentication (just use ip:port format)
-        # Understand the risk of storing password in plain-text when using proxy with authentication
+    def download_mmdb_database(self, account_id, license_key, proxy_url=None, is_ssl_verify=True):
         proxies = None
-        '''
-        proxies = {
-            "http" : "<proxy-supported-schema http|https>://<username>:<password>@<ip-address>:<port>",
-            "https" : "<proxy-supported-schema http|https>://<username>:<password>@<ip-address>:<port>"
-        }
-        '''
+        if proxy_url:
+            if proxy_url.startswith("https"):
+                proxies = {
+                    "https" : proxy_url
```

Comment:
I think we need to add both http and https with same proxy_url. Not 100% sure.

Reply:
Yes, requires both in every case, it uses https or http values based on the target URL schema.

Reply:
Done